startup/00-startup.py

```diff
@@ -18,6 +18,12 @@
 # suppress (now) spurious warnings for mpl3.3+
 mpl_qtthread.monkeypatch_pyplot()
 
+# The following code is expected to fix the issue with MPL windows 'freezing'
+#   after completion of a plan.
+from IPython import get_ipython
+ipython = get_ipython()
+ipython.run_line_magic("matplotlib", "")
```

Comment:
Such an issue was fixed for other beamlines (IOS, FMX) by commenting out the line such as in https://github.com/NSLS-II-HXN/profile_collection/blob/d30059551de204b4a4926b3f6624c7d192ecea25/ipython_config.py#L84

Reply:
I will try

Reply:
Unfortunately, commenting out `c.TerminalIPythonApp.pylab = 'auto'` does not help with MPL issue. Also there is code in HXN profile that relies on numpy being implicitly loaded in the environment and it may be a separate task to find them all.

Reply:
I fixed the the math issues, so the plans run without `c.TerminalIPythonApp.pylab = 'auto'`, but MPL issue comes back, so I will leave it uncommented for now.